transcom/mymove
# B-19460

## [B-19460t](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A932696)

## Summary

Update PPPO Base Los Angeles/Long Beach - USCG GBLOC

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

As an office user at PPPO Base Los Angeles/Long Beach - USCG

When I log into MilMove, 

Then I will see the queue's for JPPSO North Central (KKFA)




As a customer submitting a shipment for the PPPO Base Los Angeles/Long Beach - USCG

When I submit a shipment 

Then the shipment will show up in the JPPSO North Central (KKFA) queues


### Database

#### Any new migrations/schema changes:


## Screenshots
